### PR TITLE
fix(dao minimized, dao grid card): add one word labels below the icons for easier navigation (issue #102)

### PR DESCRIPTION
**Pull request by unholywhite**

### CODE REVIEW

File: astro_2.0/components/DaoDetails/DaoDetailsMinimized.module.scss [hunk 1]
```diff
@@ -42,9 +42,15 @@
 
 .controls {
   align-items: center;
+  column-gap: 2px;
```
Comment (AlexeyPonomarenko) [inline astro_2.0/components/DaoDetails/DaoDetailsMinimized.module.scss]: We might have issues with this prop in Safari. Can you check if it works there, please?

File: components/button/IconTextButton.module.scss [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  color: var(--color-primary-50);
+  column-gap: 6px;
+  cursor: pointer;
+  display: inline-flex;
+  flex-direction: column;
+  flex-shrink: 0;
+  height: 42px;
+  outline: 0;
+  padding: 0;
+  row-gap: 4px;
```
Comment (AlexeyPonomarenko) [inline components/button/IconTextButton.module.scss]: Please, verify this and column-gap works in Safari.

File: astro_2.0/components/DaoDetails/DaoDetailsMinimized.module.scss (hunk 1 above)
Comment (unholywhite) [reply astro_2.0/components/DaoDetails/DaoDetailsMinimized.module.scss]: Don't have Safari, replaced `column-gap` with `gap`.

File: components/button/IconTextButton.module.scss (hunk 2 above)
Comment (unholywhite) [reply components/button/IconTextButton.module.scss]: Replaced `row-gap` with `gap`.

**Review by VERZUOL1 — APPROVED** [review]

**Comment by RomaSha2010** [comment]
Verified on dev. Implemented

**Pull request merged**